packages/redux-dynamic-modules-react/src/DynamicModuleLoader.tsx

```diff
@@ -154,10 +160,31 @@ class DynamicModuleLoaderImpl extends React.Component<
     /**
      * Unregister sagas and reducers
      */
-    public componentWillUnmount(): void {
+    private _cleanup(): void {

```

Comment:
Just use an arrow function here?

private _cleanup = () => {
...
}

Then you can get rid of the bind call above

Reply:
Thanks, done